Cargo.toml

```diff
@@ -16,6 +16,7 @@ futures-cpupool = "0.1"
 num_cpus = "1"
 crossbeam = "0.7"
 byteorder = "1"
+ocl = "0.19"
```

Comment:
Can you make this an optional dependency, and put everything behind a `gpu` feature flag please?

Reply:
Done. Can you check if the latest commit is the best way of doing this?